core/src/commonMain/kotlin/cache/data/UserData.kt

```diff
@@ -28,7 +28,8 @@ public data class UserData(
     val bot: OptionalBoolean = OptionalBoolean.Missing,
     val publicFlags: Optional<UserFlags> = Optional.Missing(),
     val banner: String? = null,
-    val accentColor: Int? = null
+    val accentColor: Int? = null,
+    val avatarDecoration: String? = null
```

Comment:
this isn't passed in from `DiscordUser`

Reply:
Oops, I'll fix that when I get home. Is that removing this or making it an optional

Reply:
you need to pass `avatarDecoration` to this constructor in the `from` function inside the companion object. to avoid conversion, it should therefore be also an optional.